cypress/support/e2e.ts

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
+// This example support/e2e.ts is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// NOTE: There is an annoying bug https://github.com/testing-library/cypress-testing-library/issues/172
```

Comment:
Btw. based on https://github.com/testing-library/cypress-testing-library/issues/172#issuecomment-1139465640 this is maybe resolved now. Can you check this when you have time?

Reply:
I'll take a look 👍

Reply:
No such luck. Seems to still be an issue (we've got the latest `@testing-library/cypress`)

<img width="1030" alt="Screenshot 2022-07-22 at 11 20 55" src="https://user-images.githubusercontent.com/747979/180408168-6a915b88-4348-4740-af55-2e3e3c60206e.png">

Reply:
😢 Thanks for looking at this.